src/internal_modules/roc_core/target_openssl/roc_core/secure_random.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2015 Roc Streaming authors
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include "secure_random.h"
+#include "roc_core/log.h"
+#include <openssl/err.h>
+#include <openssl/rand.h>
```

Comment:
please separate local and system headers with an empty line like this:

```suggestion
#include "secure_random.h"

#include "roc_core/log.h"

#include <openssl/err.h>
#include <openssl/rand.h>
```

Reply:
Actually the first block should be:

```
#include "roc_core/secure_random.h"
#include "roc_core/log.h"
```

(we always use full path to file)